### DISCUSSION TITLE
Setup Netlify on Website branch

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/journey-book#24] [General]
Sign Up to https://app.netlify.com/
Go to https://app.netlify.com/start
Tap GitHub
Autorize Netlify to access your github
It will take you to Netlify, tap this repo name
Set branch to deploy as "website
Go down to the page and say deploy site

**Comment by Adarsh-jaiss** [comment]
![Uploading IMG_20220310_121449.jpg…]()

**Reply by KendallDoesCoding** [reply]
Think you need to wait for it to upload then comment xD

**Reply by KendallDoesCoding** [reply]
Please paste the image in, and wait till it updates, it will take 60 secs.

**Reply by KendallDoesCoding** [reply]
coz its coming as text.

**Comment by Adarsh-jaiss** [comment]
![IMG_20220310_121449](https://user-images.githubusercontent.com/96974600/157604786-e0526b0d-6072-4590-87ed-c5eb921d6514.jpg)

**Reply by KendallDoesCoding** [reply]
> ![IMG_20220310_121449](https://user-images.githubusercontent.com/96974600/157604786-e0526b0d-6072-4590-87ed-c5eb921d6514.jpg)

anything works, it doesnt really matter, ig Marketing or company site is preferable for this tho.

**Reply by Adarsh-jaiss** [reply]
what should i choose ?

**Reply by KendallDoesCoding** [reply]
I told you

> anything works, it doesnt really matter, ig Marketing or company site is preferable for this tho.

Choose Personal/Portfolio site or Marketing Company Site

**Reply by Adarsh-jaiss** [reply]
Okay so i'm gonna choose marketing or company site

**Reply by KendallDoesCoding** [reply]
Ok.

**Comment by Adarsh-jaiss** [comment]
![1646895067788510899985211515486](https://user-images.githubusercontent.com/96974600/157605456-53218acd-fc92-4b6a-9813-75d55dfbd6e1.jpg)

**Comment by Adarsh-jaiss** [comment]
Should I go for "only slect repositories"?

**Reply by KendallDoesCoding** [reply]
No, I suggest go for "All Repos" as if you ever plan on using Netlify again, it will save your time and effort.

**Reply by Adarsh-jaiss** [reply]
Okay

**Reply by Adarsh-jaiss** [reply]
So, now I have to pick up a repo

**Reply by KendallDoesCoding** [reply]
Can you send another screenshot? Or go to https://app.netlify.com/start

**Comment by Adarsh-jaiss** [comment]
So I should select journey book, right!?

**Reply by KendallDoesCoding** [reply]
Yes.

**Comment by Adarsh-jaiss** [comment]
Okay, now everything is done and i'm gonna deploy site as default

**Comment by Adarsh-jaiss** [comment]
![16468953439747526665964971684665](https://user-images.githubusercontent.com/96974600/157606102-37b1da40-c196-442b-8458-bd197d0e8157.jpg)

**Reply by Adarsh-jaiss** [reply]
Should I do it?

**Reply by KendallDoesCoding** [reply]
Nope, set branch to deploy as "website" and change later.

**Reply by KendallDoesCoding** [reply]
When we merge "website" to main, you can change it then, coz now we're working on "website" for the branch as discussed.

**Reply by Adarsh-jaiss** [reply]
So, I am gonna select website branch

**Reply by Adarsh-jaiss** [reply]
Right?

**Reply by KendallDoesCoding** [reply]
Yes. I was gonna tell this but you deleted your comment xD

> No, just tap "main" under branch to deploy and it will show website, tap website and then deploy.

**Reply by Adarsh-jaiss** [reply]
Okay, I am goona deploy it

**Reply by KendallDoesCoding** [reply]
Ok cool!

**Comment by Adarsh-jaiss** [comment]
S
![16468956239845308618238462216999](https://user-images.githubusercontent.com/96974600/157606708-429dc315-64c6-4c26-9b04-2160fcb0bd51.jpg)

**Reply by KendallDoesCoding** [reply]
nice, when it's done you will get the url, the url is the site name + .netlify.com

**Reply by Adarsh-jaiss** [reply]
Okay

**Reply by KendallDoesCoding** [reply]
When deployed 2 times, it should show:

COMING SOON on the website

![image](https://user-images.githubusercontent.com/84712013/157606954-9472af1c-55b0-45a7-8473-fb6542ff0774.png)

**Reply by Adarsh-jaiss** [reply]
Refer to #16

**Reply by KendallDoesCoding** [reply]
> S ![16468956239845308618238462216999](https://user-images.githubusercontent.com/96974600/157606708-429dc315-64c6-4c26-9b04-2160fcb0bd51.jpg)

@Adarsh-jaiss Please share the url of this page.

**Reply by Adarsh-jaiss** [reply]
It's in progress

**Reply by KendallDoesCoding** [reply]
SIte hosted at https://unruffled-chandrasekhar-ffade4.netlify.app/... Please check it out @Adarsh-jaiss

**Reply by KendallDoesCoding** [reply]
> It's in progress

It's done, it's showing my recent deploy, coming soon text.

**Reply by Adarsh-jaiss** [reply]
Yeah, website is deployed successfully 😃
